content/how-to/add-code-servers/bitbucket-cloud/access-token/index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "How to generate an app password or personal access token (PAT) in Bitbucket"
+description: "How to generate an app password or personal access token (PAT) in Bitbucket"
+lead: "How to generate an app password or personal access token (PAT) in Bitbucket"
```

Comment:
See discussion below about the `description` and `lead` metadata: https://github.com/BluBracket/docs.blubracket.com/pull/99/files#r822196764